treeherder/config/urls.py

```diff
@@ -29,5 +29,7 @@
     from graphene_django.views import GraphQLView
     from treeherder.webapp.graphql.schema import schema
     urlpatterns += [
-        url(r'^graphql', csrf_exempt(GraphQLView.as_view(graphiql=True, schema=schema))),
+        url(r'^graphql$',
```

Comment:
@camd, I don't suppose you could file the issue against [graphene-django](https://github.com/graphql-python/graphene-django) for fixing their `urlpatterns` docs? :-)